frontend/src/features/properties/components/forms/ParcelDetailForm.test.tsx

```diff
@@ -526,7 +526,7 @@ describe('ParcelDetail Functionality', () => {
     });
   });
   describe('parcel layer data tests', () => {
```

Comment:
These tests are failing because the parcel detail form no longer is displayed. Will need to migrate these tests to the new pages before the end of the sprint.